### DISCUSSION TITLE
mkdocs-material/blog/2023/10/02/sunsetting-gitter-towards-efficient-community-engagement/

### DISCUSSION THREAD

**Discussion by giscus** [squidfunk/mkdocs-material#6121] [_]
# mkdocs-material/blog/2023/10/02/sunsetting-gitter-towards-efficient-community-engagement/

Write your documentation in Markdown and create a professional static site in minutes – searchable, customizable, in 60+ languages, for all devices

https://squidfunk.github.io/mkdocs-material/blog/2023/10/02/sunsetting-gitter-towards-efficient-community-engagement/

**Comment by Andre601** [comment]
Didn't know Gitter was owned by Matrix. Nice trivia.

Tho, I feel like the Matrix Foundation isn't that good in UI design. For example did I use the element app, which apparently is created by the co-founders of matrix, and the UI is confusing and kind of a mess... So hearing Gitter having a bad UI (Which imo it does have) is not too surprising for me.

**Comment by kasdihacene** [comment]
Good job

**Comment by eliasmbd** [comment]
As an open source CM, we do seem to be facing similar challenges on our end. Thanks for sharing this rationale and hopefully it has been working out for you.
